aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,197 @@
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication().AddJwtBearer();
+
+var app = builder.Build();
+
+app.Run();
+```
+
+By default, the  [`WebApplication`](/dotnet/api/microsoft.aspnetcore.builder.webapplication) automatically registers the authentication and authorization middlewares if certain authentication and authorization services are enabled. In the following code sample below, it's not necessary to invoke [`UseAuthentication`](/dotnet/api/microsoft.aspnetcore.builder.authappbuilderextensions.useauthentication) or [`UseAuthorization`](/dotnet/api/microsoft.aspnetcore.builder.authorizationappbuilderextensions.useauthorization) to register the middlewares since [`WebApplication`](/dotnet/api/microsoft.aspnetcore.builder.webapplication) does this automatically after services are added.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication().AddJwtBearer();
+builder.Services.AddAuthorization();
+
+var app = builder.Build();
+
+app.Run();
+```
+
+In some cases, such as controlling middleware order, it's necessary to explicitly register authentication and authorization. In the following sample, the authentication middleware runs _after_ the CORS middleware has run.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddAuthentication().AddJwtBearer();
+builder.Services.AddAuthorization();
+
+var app = builder.Build();
+
+app.UseCors();
+app.UseAuthentication();
+app.UseAuthorization();
+
+app.Run();
+```
+
+### Configuring authentication strategy
+
+Authentication strategies typically support a variety of configurations that are loaded in via options. Minimal application support loading options from configuration for the following authentication strategies:
+
+- JWT bearer-based authentication strategies
+- OpenID Connection-based authentication strategies
+
+The ASP.NET framework expects to find these options under the `Authentication:Schemes:{SchemeName}` section in configuration. The `appsettings.json` definition below defines two different schemes, `Bearer` and `LocalAuthIssuer`, with their respective options. The `Authentication:DefaultScheme` option can be used to configure the default authentication strategy that will be used.
+
+```json
+{
+  "Authentication": {
+    "DefaultScheme":  "LocalAuthIssuer",
+    "Schemes": {
+      "Bearer": {
+        "ValidAudiences": [
+          "https://localhost:7259",
+          "http://localhost:5259"
+        ],
+        "ValidIssuer": "dotnet-user-jwts"
+      },
+      "LocalAuthIssuer": {
+        "ValidAudiences": [
+          "https://localhost:7259",
+          "http://localhost:5259"
+        ],
+        "ValidIssuer": "local-auth"
+      }
+    }
+  }
+}
+```
+
+In `Program.cs`, we register two JWT bearer-based authentication strategies: one with the default scheme name ("Bearer") and one with the "LocalAuthIssuer" scheme name. 
```

Comment:
This is a little confusing, we just explained setting a default scheme and used "LocalAuthIssuer" as the default, but now we're claiming the default scheme is "Bearer".

Reply:
Ah good insight -- I'll reword this to make it clear we are overriding the framework default.